### PR TITLE
Euca 4.2 merge 4.1

### PR DESCRIPTION
**Pull request by viglesiasce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +109,7 @@ Some common attributes are:
     <td><tt>["eucalyptus"]["source-branch"]</tt></td>
     <td>String</td>
     <td>Branch to use when building from source</td>
-    <td><tt>testing</tt></td>
+    <td><tt>maint-4.1</tt></td>
```
Comment (tbeckham) [inline README.md]: Since this is merge to 4.2 do we want this as master instead?

File: Vagrantfile [hunk 2]
```diff
@@ -13,9 +13,9 @@ Vagrant.configure("2") do |config|
       chef.roles_path = "roles"
       chef.add_role "cloud-in-a-box"  
       chef.json = { "eucalyptus" => { ## Choose whether to compile binaries from "source" or "packages"
-                                      "install-type" => "packages",
+                                      "install-type" => "source",
                                       ## Does not change package version, use "eucalyptus-repo" variable
-                                      "source-branch" => "testing",
+                                      "source-branch" => "maint-4.1",
```
Comment (tbeckham) [inline Vagrantfile]: same comment here, should this be master?

File: attributes/default.rb [hunk 3]
```diff
@@ -1,10 +1,10 @@
 #### Install Info
 default["eucalyptus"]["install-type"] = "packages"
 default["eucalyptus"]["source-repo"] = "https://github.com/eucalyptus/eucalyptus.git"
-default["eucalyptus"]["source-branch"] = "testing"
+default["eucalyptus"]["source-branch"] = "maint-4.1"
```
Comment (tbeckham) [inline attributes/default.rb]: same.

**Comment by viglesiasce** [comment]
@tbeckham we need to make all the changes you proposed. Merging this first is the right step, since in most cases the euca-4.2 was pointing at non-existent branches not just the wrong ones.

**Pull request merged**